primitives/src/locktime/absolute.rs

```diff
@@ -108,7 +108,6 @@ impl LockTime {
     ///
     /// ```rust
     /// # use bitcoin_primitives::absolute::LockTime;
-    /// # let n = LockTime::from_consensus(741521); // n OP_CHECKLOCKTIMEVERIFY
```

Comment:
OT, bother! that comment was certainly supposed to add additional useful information. Flagging so that the poor soul who does #3334 has the information that was lost.

Reply:
I'm unsure what you mean. This line is hidden and the variable it assigns isn't used.

Reply:
Oh, I meant that when I wrote these comments originally I tried to add additional information to the `n` variable by documenting with `// n OP_CHECKLOCKTIMEVERIFY` - that's all. (And I flagged it because when, I assume me, goes to improve the locktime API and fix the docs I might get confused.